system/Packages/perc.widget.imageSlider/sys__UserDependency--rxconfig/Widgets/percImageSlider.xml

```diff
@@ -563,10 +580,13 @@
                   <$perc_slide_caption_2_tag_type class="perc-slide-image-caption-2 perc-slide-caption-horizontal-${perc_slide_caption_alignment} $captionclass">${image.imageCaption2}</$perc_slide_caption_2_tag_type>
                 </div>
                 #end##
-                #if( $image.clickableLink != "" )##
-                <a href="$image.clickableLink" target="_blank" rel = "noopener noreferrer">
-                #end##
+
+				#if( $perc_slide_image_link == "false" )##
```

Comment:
@mario-percussion Won't this only render the a tag around the image if the user unchecks images are links option.  This seems backwarns.

Reply:
Yeah you're right, just changed it